src/main/scala/ai/privado/audit/APIReport.scala

```diff
@@ -0,0 +1,46 @@
+package ai.privado.audit
+
+import ai.privado.cache.RuleCache
+import ai.privado.model.{CatLevelOne, Constants}
+import io.shiftleft.codepropertygraph.generated.Cpg
+import org.slf4j.LoggerFactory
+
+import scala.collection.mutable.ListBuffer
+import io.shiftleft.semanticcpg.language.*
+
+import scala.util.{Success, Try}
+
+object APIReport {
+
+  private val logger = LoggerFactory.getLogger(getClass)
+
+  def processAPIAudit(xtocpg: Try[Cpg], ruleCache: RuleCache): List[List[String]] = {
+    val COMMON_IGNORED_SINKS_REGEX        = ruleCache.getSystemConfigByKey(Constants.ignoredSinks)
+    val APISINKS_REGEX                    = ruleCache.getSystemConfigByKey(Constants.apiSinks)
+    val COMMON_HTTP_PACKAGE_REGEX: String = ruleCache.getSystemConfigByKey(Constants.apiHttpLibraries)
+
+    val workFlowResult = new ListBuffer[List[String]]
+    xtocpg match {
+      case Success(cpg) => {
+        val cacheCall = cpg.call.where(_.nameNot("(<operator|<init).*")).l
+        val apis = cacheCall
```

Comment:
@ankit-privado This will yield the same result as we have for tagging, How does this help for auditing??

Here in audit we plan to get occurrences which look like a HTTP call but haven't been tagged yet.
Example `client.post`, if this is not tagged this should go to audit report

Reply:
@khemrajrathore okay got you. thanks

Reply:
done.